examples/cloudaccounts/k8s/README.md

```diff
@@ -53,6 +53,36 @@ func main() {
 	}
 	fmt.Printf("response type: %T\n Content: %+v\n", OrganizationalUnitIDResponse, OrganizationalUnitIDResponse) 
 
+	// Enable Runtime Protection
```

Comment:
I am not sure we want to have runtime here, since it is still in PP and should not be visible to customers. please validate with Yonatan

Reply:
We verified with Andrew and he wanted to add RP as well